<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CwmsRightsManager</class>
 <widget class="QDialog" name="CwmsRightsManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rechteverwaltung</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QLabel" name="m_pqlArticleType">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(232, 232, 232, 255));</string>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:15pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Rechteverwaltung&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Gruppen</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QListWidget" name="m_pqlbGroups"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="m_pqpbAddGroup">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../WMSadmin/resources/emresources.qrc">
            <normaloff>:/icons/arrow-right24.png</normaloff>:/icons/arrow-right24.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Alt+H</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="spacer3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>111</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Anwender</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QListWidget" name="m_pqlbUsers"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="m_pqpbAddUser">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../WMSadmin/resources/emresources.qrc">
            <normaloff>:/icons/arrow-right24.png</normaloff>:/icons/arrow-right24.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Alt+H</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>284</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Rechte</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QTreeWidget" name="m_pqlvRights">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Typ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Recht</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="m_pqpbRemoveUser">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../WMSadmin/resources/emresources.qrc">
           <normaloff>:/icons/arrow-left24.png</normaloff>:/icons/arrow-left24.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>361</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbClose">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../WMSadmin/resources/emresources.qrc">
         <normaloff>:/icons/ok24.png</normaloff>:/icons/ok24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Alt+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../WMSadmin/resources/emresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pqpbAddUser</sender>
   <signal>clicked()</signal>
   <receiver>CwmsRightsManager</receiver>
   <slot>AddUserClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbAddGroup</sender>
   <signal>clicked()</signal>
   <receiver>CwmsRightsManager</receiver>
   <slot>AddGroupClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>547</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbRemoveUser</sender>
   <signal>clicked()</signal>
   <receiver>CwmsRightsManager</receiver>
   <slot>RemoveRightClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqlvRights</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>CwmsRightsManager</receiver>
   <slot>RightsListViewDoubleClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>854</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqlbGroups</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>CwmsRightsManager</receiver>
   <slot>AddGroupClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqlbUsers</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>CwmsRightsManager</receiver>
   <slot>AddUserClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbClose</sender>
   <signal>clicked()</signal>
   <receiver>CwmsRightsManager</receiver>
   <slot>CloseClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>666</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>838</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddGroupClickedSlot()</slot>
  <slot>RemoveRightClickedSlot()</slot>
  <slot>RightsListViewDoubleClickedSlot()</slot>
  <slot>AddUserClickedSlot()</slot>
  <slot>CloseClickedSlot()</slot>
 </slots>
</ui>
